test/mixed_workload_test.go

```diff
@@ -0,0 +1,108 @@
+package test
```

Comment:
Perhaps also rename file to something `test/admission_test.go`?

Reply:
Done